src/cobra_db/mongo_dao.py

```diff
@@ -297,8 +296,7 @@ def get_by_id(self, _id: Id, obj=True) -> Union[Patient, dict]:
 
 class StudyDao(EntityDao):
     '''
-    Get / insert datainfo in Study level
-    Each Study is the Series with same Patient_Date
+    Get or insert study from / to the database
     (Manages access to the Study collection in MongoDB)
```

Comment:
The collection is named after the class that it holds so it is "RadiologicalStudy" not just Study